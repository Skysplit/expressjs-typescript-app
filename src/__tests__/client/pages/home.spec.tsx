import React from 'react';
import HomePage from '@client/pages/home';
import { shallow } from 'enzyme';

describe('pages/home', () => {
  test('should render component', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
