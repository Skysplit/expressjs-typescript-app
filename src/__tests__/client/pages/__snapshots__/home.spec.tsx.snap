// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/home should render component 1`] = `
<React.Fragment>
  <h1>
    Hello world
  </h1>
  <LinkRoutes
    route="test"
  >
    <a>
      Go to some page
    </a>
  </LinkRoutes>
</React.Fragment>
`;
